tests/core/channels/test_twilio_voice.py

```diff
@@ -0,0 +1,206 @@
+import json
```

Comment:
it would be really cool if we could also test the `receive` function of the webhook. Especially the parts which re-utter the last bot utterances seem something which would better be tested.

Reply:
@wochinge I had tried doing this replicating this Google Hangouts test, but I wasn't able to get it to work. Is this the proper approach to building a test for this?

https://github.com/RasaHQ/rasa/blob/main/tests/core/channels/test_hangouts.py#L126

Reply:
```python
async def test_twilio_receive(stack_agent: Agent):
    app = server.create_app(agent=stack_agent)

    inputs = {
        "initial_prompt": "hello",
        "reprompt_fallback_phrase": "i didn't get that",
        "speech_model": "default",
        "speech_timeout": "5",
        "assistant_voice": "woman",
        "enhanced": "false",
    }

    tv = TwilioVoiceInput(**inputs)
    channel.register([tv], app, "/webhooks/")

    client = app.asgi_client

    body = {"From": "Tobias", "SpeechResult": None, "CallStatus": "ringing"}
    _, response = await client.post(
        "/webhooks/twilio_voice/webhook",
        headers={"Content-type": "application/x-www-form-urlencoded"},
        data=body,
    )
    assert response.status == 200
    # Actual test xml content
    assert response.body
```

This should work. I think that would be super cool because then you can test the entire flow

Reply:
Neat! This is good to know for the future.